includes/discovery/processors/cambium.inc.php

```diff
@@ -21,7 +21,7 @@
             '0',
             'cambium-cpu',
             $descr,
-            '100',
+            '10',
```

Comment:
Just confirmed this on my own install, removing the file entirely works great. Will update this :)